src/csharp/Microsoft.Spark.Extensions.Delta/Tables/DeltaTable.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Extensions.Delta.Tables
+{
+    /// <summary>
+    /// Main class for programmatically interacting with Delta tables.
+    /// You can create DeltaTable instances using the static methods.
+    /// 
+    /// <code>
+    /// DeltaTable.ForPath(sparkSession, pathToTheDeltaTable)
+    /// </code>
+    /// </summary>
+    public class DeltaTable : IJvmObjectReferenceProvider
+    {
+        private readonly JvmObjectReference _jvmObject;
```

Comment:
```suggestion
        private static IJvmBridge Jvm { get; } = SparkEnvironment.JvmBridge;
        private static readonly string s_deltaTableClassName = "io.delta.tables.DeltaTable";
            
        private readonly JvmObjectReference _jvmObject;
```

Reply:
Why?

Reply:
This is the pattern we use.  For example [Functions.cs](https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark/Sql/Functions.cs) and [Window.cs](https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark/Sql/Expressions/Window.cs)

Reply:
Those are static classes.

Reply:
Sure, we can leave it as is then.